src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -72,6 +79,7 @@ public function getFunctions()
         return array(
             new \Twig_SimpleFunction('profiler_dump', $profilerDump, array('is_safe' => array('html'), 'needs_environment' => true)),
             new \Twig_SimpleFunction('profiler_dump_log', array($this, 'dumpLog'), array('is_safe' => array('html'), 'needs_environment' => true)),
+            new \Twig_SimpleFunction('help_attrs', array($this, 'help'), array('is_safe' => array('html'))),
```

Comment:
This should be prefixed with `profiler_` to prevent conflicts with other extensions.